status/statusgo_backend_new/chat.nim

```diff
@@ -1,13 +1,66 @@
-import json
+import json, sequtils, sugar
 import core, utils
 import response_type
 
 export response_type
 
+proc saveChat*(
+    chatId: string,
+    chatType: int,
+    active: bool = true,
+    color: string = "#000000",
+    ensName: string = "",
+    profile: string = "", 
+    joined: int64 = 0
+    ) {.raises: [Exception].} =
+  # TODO: ideally status-go/stimbus should handle some of these fields instead of having the client
+  # send them: lastMessage, unviewedMEssagesCount, timestamp, lastClockValue, name?
+  discard callPrivateRPC("saveChat".prefix, %* [
```

Comment:
@jrainville don't add any logic to `status-lib`, just return `RpcResponse[JsonNode]` to the app, we return what is received from the `status-go` part and let the app (service there) determine what to do next.

Reply:
My bad, it's fixed now @saledjenic